VoodooI2CGoodix/Info.plist

```diff
@@ -53,8 +53,6 @@
 		<string>1.4</string>
 		<key>com.apple.iokit.IOHIDFamily</key>
 		<string>2.0</string>
-		<key>com.apple.iokit.IOGraphicsFamily</key>
-		<string>1.0.0b1</string>
```

Comment:
Is it the mere presence of the dependency, or it’s usage that causes the failure on Big Sur? Trying to ascertain whether we can keep the dep and choose what to use in code, keeping support for older macOS, or whether we need a branch to maintain support for older versions.

Reply:
i havent tested keeping the dependency and making the code changes only but i will in a lil while...the kext wouldnt even attach on big sur until i removed the dependency in the plist and i got the idea from alterations made to the voodooi2chid satellite

Reply:
also...it could be a simple sdk version target issue(set it to 15) so im gonna attempt with 14 and report back

Reply:
good news...seems we just needed to build it with catalina(and possibly lower the target version) but now this one version works on both. just in case its a fluke...will try to wait until @balopez confirms so by the time he replies hopefully u will be ready to review everything

Reply:
2 more things if u get motivated enough to take a look...double click and disable when lid closed. its possible the code is identical to the touchscreen files within voodooi2chid just like the fix for this was...gonna mess around with it for a while myself just to see

Reply:
just wanted to add...found out that 2..5.2 in fact works with a warm boot from windows/linux at least on big sur(will come back and confirm catalina but last i looked it wasnt even attaching)